### PR TITLE
Added symfony/var-dumper requirement

### LINKED ISSUE #15: Add symfony/var-dumper as a requirement?
### What steps will reproduce the problem?

In this class: https://github.com/yiisoft/yii2-shell/blob/master/YiiCaster.php you import a Symfony class: `use Symfony\Component\VarDumper\Caster\Caster;` but you don't require `symfony/var-dumper` in the `composer.json` of this project: https://github.com/yiisoft/yii2-shell/blob/master/composer.json

This works by chance and pure luck. Some package is installing the package you need (`symfony/var-dumper`) so this works without you actually requiring that package.

### What's expected?

`symfony/var-dumper` should be added to the `require` section of the `composer.json` file.

### What do you get instead?

-

### Additional info

| Q                | A
| ---------------- | ---
| Yii vesion       | Any
| PHP version      | Any
| Operating system | Any

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #15

This tries to fix #15. I don't know which is the policy of this project about dependency versions, so I just followed the same done here: https://github.com/yiisoft/yii2-queue/blob/fe279cbfdf513ba55e52193da374403c83d3c596/composer.json#L21

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     ],
     "require": {
         "yiisoft/yii2": "~2.0.0",
-        "psy/psysh": "~0.8.1"
+        "psy/psysh": "~0.8.1",
+        "symfony/var-dumper": "*"
```
Comment (schmunk42) [inline composer.json]: We should actually not use an unbound version constraint. But on the other hand ... they released a new version for three major versions 17 days ago :)

Comment (samdark) [inline composer.json]: I think we need to specify a version there.

Comment (schmunk42) [reply composer.json]: Like so `2.* || 3.* || 4.*`?

Comment (samdark) [reply composer.json]: ~2.7|~3.0|~4.0

Comment (samdark) [reply composer.json]: ~2.7|~3.0|~4.0

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**